docs/book/src/networking-introduction.md

```diff
@@ -0,0 +1,123 @@
+The protocol is [reliable ordered][guarantees] in nature.
+
+**UDP** stands for 'user datagram protocol' and adds certain guarantees ontop of [IP][3], but unlike TCP, 
+instead of adding lots of features and complexity, UDP is a very thin layer over IP and is also [unreliable][guarantees] in nature.
+
+| Feature |  TCP  | UDP | QUIC
+| :-------------: | :-------------: | :-------------: | :-------------: |
+|  [Connection-Oriented][6]           |       Yes                      | No                       | Yes
+|  [Transport Guarantees][guarantees] | [Reliable Ordered][guarantees] | [Unreliable][guarantees] | [Reliable Ordered][guarantees] and [Unreliable][guarantees] 
+|  Packet Transfer                    | [Stream-based][4]              | Message based            | Message based and Stream based
+|  Automatic [fragmentation][8]       | Yes                            | Yes, but better to stay below datagram size limit | Yes
+|  Header Size                        |  20 bytes                      | 8 bytes                  |  16 bytes  
+|  [Control Flow, Congestion Avoidance/Control][5] | Yes               | No                       |  Yes, and user controlled                                          
+
+## 2. The 5 Transport Guarantees
+
+There are 5 main different ways you can transfer data:
+
+| Transport Guarantees         | Packet Drop [(1)][1]  | Packet Duplication [(2)][2] | Packet Order | Packet Delivery |
+| :-------------:              | :-------------: | :-------------: | :-------------: | :-------------:
+|   **Unreliable Unordered**   |       Any       |      Yes        |     No          |   No
+|   **Unreliable Sequenced**   |    Any + old    |      No         |     Sequenced   |   No
+|   **Reliable Unordered**     |       No        |      No         |     No          |   Yes
+|   **Reliable Ordered**       |       No        |      No         |     Ordered     |   Yes
+|   **Reliable Sequenced**     |    Only old     |      No         |     Sequenced   |   Only newest
+
+Unreliability gives great uncertainty with a lot of freedom, while reliability gives great certainty with costs in speed and freedom.
+That is why protocols such as QUIC, RUDP, SCTP, QUIC, netcode, laminar are built on UDP instead of TCP. 
+UDP gives the end-user more control over the transmission than TCP can do. 
+
+## 3. Issues with TCP 
+
+Now the golden question: "Why choose so much uncertainty with UDP when TCP is so reliable and safe?". 
+To answer that question we will have to delve a little deeper into some issues with TCP. 
+
+### Head-of-line Blocking
+
+One of the biggest issues with TCP is that of Head-of-line blocking. 
+It is a convenient feature because it ensures that all packages are sent and arrive in [order][guarantees]. 
+However, in cases of high throughput (multiplayer game networking) and big load in a short time (web page load), this can be catastrophic to your application performance.
```

Comment:
This doesn't mention multiplexing yet. I think multiplexing is pretty essential to the concept of HoL blocking.

Reply:
See the web section